jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/inputGroup/InputGroupCustomFileInputTests.java

```diff
@@ -0,0 +1,170 @@
+package io.github.epam.bootstrap.tests.composite.section.inputGroup;
+
+import io.github.epam.TestsInit;
+import org.openqa.selenium.By;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static com.epam.jdi.light.driver.get.DriverData.PROJECT_PATH;
+import static com.epam.jdi.tools.PathUtils.mergePath;
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.customFileInputPrepend;
+import static io.github.com.pages.BootstrapPage.customFileInputAppend;
+import static io.github.epam.bootstrap.tests.BaseValidations.baseValidation;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+import static org.hamcrest.Matchers.is;
+
+/**
+ * Created by Olga Ivanova on 19.09.2019
+ * Email: olga_ivanova@epam.com
+ */
+
+public class InputGroupCustomFileInputTests extends TestsInit {
+    public final static String LABEL_TEXT = "Upload";
```

Comment:
it's more common way to write static at first (if it's interesting to you: https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers).